grid-demo/src/app/angular/row-dom-performance/custom-clr-virtual-rows.directive.ts

```diff
@@ -210,6 +221,16 @@ export class CustomClrVirtualRowsDirective<T> implements OnInit, DoCheck, OnDest
   private restoreActiveCellAfterRenderedRangeUpdate() {
     if (this.activeCellElement && this.activeCellElement.tabIndex === 0) {
       setTimeout(() => {
+        this.updateAriaRowIndexes();
```

Comment:
Is this a separate bug fix?

Reply:
no, it's only necessary for this feature